04/app.js

```diff
@@ -2,8 +2,28 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '27',
+        month: '05',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+const now = new Date();
+
+function checkBirthday() {
+    if ((now.getDate() === parseInt(user['born']['day'])) && ((now.getMonth() + 1) === parseInt(user['born']['month']))) { // czy tutaj można było pominąć parseInt() i użyć "==" ?
```

Comment:
Lepsza praktyką jest używanie `===`.

PS. Dużo dzieje się  w tym `if`-ie - postaraj się to zapisać czytelniej. Możesz użyć dodatkowych zmiennych.

Reply:
Zmienione